config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
These are not RESTful you can read more about RESTful APIs [here](https://www.restapitutorial.com/lessons/httpmethods.html#:~:text=The%20primary%20or%20most-commonly,but%20are%20utilized%20less%20frequently.)
Let's also use the [`resources`](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router) to generate our routes